test/fixtures/component-playground/proxies.js

```diff
@@ -29,6 +29,12 @@ const MarkupProxy = React.createClass({
 });
 
 module.exports = _.merge({}, selectedFixture, {
-  proxies: [PropMutatorProxy, MarkupProxy],
+  proxies: [{
+    component: PropMutatorProxy,
+    props: {},
```

Comment:
> That being said, this means you can only send the contents of the store once to the ReduxProxy, global for all component/fixtures loaded? Is this correct?

Yes, you're right, this is a problem.

> If yes, I suggest the following alternative: Revert the props thing and continue to send the entire fixture to all proxies.

If we do this, aren't we decoupling props from the proxies too much?
What happens if another proxy needs a key named 'store' as well, for completely different reasons? It will use the same value.